### DISCUSSION TITLE
Semantic Color Usage

### DISCUSSION THREAD

**Discussion by Courtny** [microsoft/atlas-design#105] [Component]
You can use semantic colors to visualize the status or state of business data.

Semantic colors denote standard value states (such as good, bad, or warning). Each color has the same basic meaning in all contexts. Nearly all input controls **_could_** support semantic colors.

Semantic colors will mostly be seen used for notification banners and form erroring across the system.

![Image of Semantic Colors](https://ucd5bc0608a1fcd540dad94db4af.previews.dropboxusercontent.com/p/thumb/ABL6uLwO-FaujBdMG2iqldKbtGmNEgnmC8_V8ITNj93rY-cRt9r8AyRCNBRFHJHLCI4GQVEuN9cQH5q3xMx0fHDr3YvoSaq9q1PVFvHnnDIFeleOJepi8AIuct9UcQ2nksrcjaiHaPICuPWjst8a_5LpHxvvffLtJPKYBSK4op9dFDdom44n8Y47hXvdTrrbNMQFCrXTSoWqRLjXu7sRHS6-1SQZyZnaMXTx4C2JdOiqO-wtSt_vT4l1Tag42Ju5hjCJGN_orK7-DcjFbedXISFbsJaQ-LPkJ9b2FlC0UihV-92UVK0QerOlBmNn77cQ0lG1HaBqGpTVf1QlkSNiv6EEcE8-Ymt2qXcHz0UTc1VTwrPQeTRhRDJVXAaHxMR_Jc6by_f5Kc9bMMvPMY49fTqb/p.png?fv_content=true&size_mode=5)

**Comment by Courtny** [comment]
@wibjorn We should discuss whether we'd like to support light blue banners.

**Reply by wibjorn** [reply]
All the base version of the theme colors should be fine as backgrounds so long as they are using their invert color as the text.

`.background-color-primary` (background color)
`.color-primary-invert` (text color)

There's a change we could run into some complexities if we're expecting multiple button colors or other elements on top of them.

**Comment by wibjorn** [comment]
Currently the names of our semantic colors are as follows (I've put their color on light theme in parentheses).

- primary (blue)
- secondary (grey)
- tertiary (dark navy)
- success (green)
- info (purple)
- warning (yellow)
- danger (red)

**Comment by Courtny** [comment]
Current guidance for semantic coloring: 

![Semantic-Color-August-30-2021](https://user-images.githubusercontent.com/1640554/131410937-6586c494-55d0-4764-a9e4-6f475bd03bbd.png)
